tests/test_runner.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import warnings
+from memo import memlist, Runner, grid
+
+
+@pytest.mark.parametrize("kw", [{"backend": "loky", "n_jobs": -1}, {"backend": "threading", "n_jobs": -1}, {"backend": "multiprocessing", "n_jobs": -1}])
```

Comment:
Since `n_jobs` is constant in all these examples, do we really need it to be parametrized?

Reply:
Good point. This was a side effect of me copy and pasting at midnight.

Reply:
we could just define n_jobs in the test function instead. That way you don't have it repeated 3 times.